release/models/aft/openconfig-aft-common.yang

```diff
@@ -244,11 +258,156 @@ submodule openconfig-aft-common {
+              in the list is the inner-most packet at the bottom of the
+              stack.
+
+              For example, in an encapsulation stack with MPLS in UDP, the
+              first index in the list is the MPLS packet and the second
+              index is UDP.";
+
+            key "index";
+
+            leaf index {
+              type leafref {
+                path "../state/index";
+              }
+              description
+                "A unique index identifying an encapsulation header in a stack
+                of encapsulation headers.";
+            }
+
+            container state {
+              description
+                "State parameters relating to encapsulation headers.";
+              uses aft-common-nexthop-encap-headers-state;
+            }
+
+            container gre {
+              when "../state/type = 'oc-aftt:GRE'";
+              description
+                "Container of nodes for GRE encapsulation.";
+              container state {
+                description
+                  "State parameters relating to GRE encapsulation headers.";
+                uses aft-common-entry-nexthop-gre-state;
+              }
+            }
+
+            container ipv4 {
+              when "../state/type = 'oc-aftt:IPV4'";
+              description
+                "Container of nodes for UDP in IPv4 encapsulation.  When this
+                container is used, an IPv4 packet with no transport header
+                is added to the encapsulation list.";
+              container state {
+                description
+                  "State parameters relating to IP encapsulation headers.";
+                uses aft-common-entry-nexthop-ip-state;
+              }
+            }
+
+            container ipv6 {
+              when "../state/type = 'oc-aftt:IPV6'";
+              description
+                "Container of nodes for UDP in IPv6 encapsulation.  When this
+                container is used, an IPv6 packet with no transport header
+                is added to the encapsulation list.";
+              container state {
+                description
+                  "State parameters relating to IP encapsulation headers.";
+                uses aft-common-entry-nexthop-ip-state;
+              }
+            }
+
+            container mpls {
+              when "../state/type = 'oc-aftt:MPLS'";
+              description
+                "Container of nodes for MPLS encapsulation.";
+              container state {
+                description
+                  "State parameters relating to MPLS encapsulation headers.";
+                uses aft-common-entry-nexthop-mpls-state;
+              }
+            }
+
+            container udp-v4 {
+              when "../state/type = 'oc-aftt:UDPV4'";
+              description
+                "Container of nodes for UDP in IPv4 encapsulation.  When this
+                container is used, an IPv4 packet with a UDP header is added
```

Comment:
```suggestion
                container is used, an IPv4 header and a UDP header is added
```

Suggest this edit here and below. We're not adding a new packet right?

Reply:
done